beaver/src/components/header.js

```diff
@@ -85,10 +83,7 @@ const Header = ( { closeModal, getOrder, getSearchQuery } ) => {
 							/>
 						) }
 
-						{ window.tiTpc.postTypes.includes(
-							window.tiTpc.postType
-						) &&
-							parseInt( window.tiTpc.tier ) === 3 && (
+						{ window.tiTpc.postTypes.includes(window.tiTpc.postType) && (
```

Comment:
One small suggestion here: you can declare this as a constant outside the component with a name like `isPostTypeCompatible` to highlight the compatibility check.